### PR TITLE
Tammy/enable moderate amplifications test

### PR DESCRIPTION
**Pull request by TammushB**

### CODE REVIEW

File: R/cn.mops.R [hunk 1]
```diff
@@ -509,6 +511,8 @@ cn.mops <- function(input,I = c(0.025,0.5,1,1.5,2,2.5,3,3.5,4),
 				parallel::stopCluster(cl)
 			}
 			
+			#browser()
```
Comment (ilyasoifer) [inline R/cn.mops.R]: Comments

File: R/cn.mops.R [hunk 2]
```diff
@@ -249,7 +249,9 @@ cn.mops <- function(input,I = c(0.025,0.5,1,1.5,2,2.5,3,3.5,4),
 		norm=1, normType="poisson",sizeFactor="mean",normQu=0.25, quSizeFactor=0.75,
 		upperThreshold=0.5,lowerThreshold=-0.9,
 		minWidth=3,segAlgorithm="fast",minReadCount=5,useMedian=FALSE,
-		returnPosterior=FALSE,...){
+		returnPosterior=FALSE,moderate_amplifications=FALSE,...){
+
```
Comment (ilyasoifer) [inline R/cn.mops.R:253]: Comments

File: R/cn.mops.R [hunk 3]
```diff
@@ -564,18 +568,37 @@ cn.mops <- function(input,I = c(0.025,0.5,1,1.5,2,2.5,3,3.5,4),
...
-					parallel::clusterEvalQ(cl,"segment")
-					resSegmList[[chrom]] <- parallel::parApply(cl,sINI[chrIdx, ,drop=FALSE],2,
-							segment,minSeg=minWidth,...)
-					parallel::stopCluster(cl)
+					
+					if(moderate_amplifications){
+						cl <- parallel::makeCluster(as.integer(parallel),type="SOCK")
+						parallel::clusterEvalQ(cl,"segment")
+						resSegmList[[chrom]] <- parallel::parApply(cl,sINI[chrIdx, ,drop=FALSE],2,
+								segment,minSeg=minWidth,alpha=0.065,segMedianT=c(0.25,-0.5),...)
```
Comment (ilyasoifer) [inline R/cn.mops.R:587]: is the change to alpha really necessary?

Comment (TammushB) [reply R/cn.mops.R:587]: yes. this is necessary for this line: https://github.com/Ultimagen/cn.mops/blob/ba697e55bca62962c3ad12f588d2cca7beaa793e/R/segment.R#L111 
to lower down the cutoff to consider segments as potential CNVs.

**Comment by TammushB** [comment]
merging this.

**Pull request merged**